webargs_starlette/dict2schema.py

```diff
@@ -0,0 +1,19 @@
+import typing
+
+import marshmallow as ma
+
+
+def dict2schema(dct, *, schema_class=ma.Schema) -> typing.Type[ma.Schema]:
```

Comment:
This shouldn't be necessary, since webargs-starlette relies on marshmallow 3, which already has `Schema.from_dict`.

Reply:
done (changed call in `annotations2schema`)